client/driver/qemu.go

```diff
@@ -464,6 +464,7 @@ func (h *qemuHandle) Kill() error {
 	// If Nomad did not send a graceful shutdown signal, issue an interrupt to
 	// the qemu process as a last resort
 	if gracefulShutdownSent == false {
+		h.logger.Printf("[DEBUG] driver.qemu: graceful shutdown is not enabled, sending an interrupt signal to QEMU")
```

Comment:
It's nice to include something to indicate what task log lines refer to. Perhaps something like:

> [DEBUG] driver.qemu: graceful shutdown disabled, sending interrupt signal to pid %d", h.userPid